spec/test-data/translations/glotpress-download/stubs/fr.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Translation-Revision-Date: 2021-03-31 09:54:04+0000
+Plural-Forms: nplurals=2; plural=n > 1;
+Generator: GlotPress/2.4.0-alpha
+Language: fr
+-->
+<resources>
+	<string name="product_attributes_error_saving">Erreur lors de l’enregistrement de vos attributs</string>
```

Comment:
This is the network stub for the content as it is received when downloaded by glotpress, before being post-processed by our action. Those files use tab as indentation (while our final files use 4 spaces).